src/sagemaker/estimator.py

```diff
@@ -852,6 +872,12 @@ def _prepare_for_training(self, job_name=None):
 
             code_dir = 'file://' + self.source_dir
             script = self.entry_point
+        elif self.enable_network_isolation() and self.entry_point:
+            relative_code_location = 'input/data/code'
+            self.uploaded_code = self._stage_user_code_in_s3(tar=False, s3_location=relative_code_location)
+            code_dir = "/opt/ml/{}".format(relative_code_location)
```

Comment:
Unsure about this line, I feel like it should not be hardcoded to `/opt/ml/...." here